Grades.java

```diff
@@ -13,6 +15,51 @@
 public class Grades {
 
     public static void main(String[] args) {
-        // enter your code here :)
+
+        Scanner input = new Scanner(System.in);
+        System.out.printf("Please enter the number of students: ");
+        int numOfStudents = input.nextInt();
+
+        if (numOfStudents <= 0) {
+            System.out.println("The number of students must be positive.");
+            return;
+        }
+
+        byte studentsArray[] = new byte[numOfStudents];
+        byte indexHighest = 1;
+        byte indexLowest = 1;
+
+        int highest = 0;
+        int lowest = 100;
+        int sum = 0;
+
+        System.out.println("Please enter the grades one by one: ");
+
+        for (byte i=0 ; i < studentsArray.length ; i++) {
+
+            int grade = input.nextInt();
```

Comment:
The grade should be of type byte and not int as suggested by the hint in the comments

Reply:
Indeed i got the hint, but I had to set the grade as an int or otherwise the program will terminate if someone enters an out of range value.

Reply:
Well at this point we count on the user to do as he/she's told.. so this argument can also rise when dealing with non-integer values... No limit for perfection :)
Still this is a NON-issue - just a small comment.
When we deal with exceptions, you'll know how to handle this perfectly (maybe you already know :) )

Reply:
Yes, I know a bit about try..catch blocks but not too much.
Ok, i'll revise. Thank you :-)